tls/s2n_crypto.h

```diff
@@ -64,6 +64,7 @@ struct s2n_crypto_parameters {
     uint8_t server_implicit_iv[S2N_TLS_MAX_IV_LEN];
     uint8_t client_app_secret[S2N_TLS13_SECRET_MAX_LEN];
     uint8_t server_app_secret[S2N_TLS13_SECRET_MAX_LEN];
+    uint8_t resumption_master_secret[S2N_TLS13_SECRET_MAX_LEN];
```

Comment:
I'm actually wondering if the resumption_master_secret actually belongs on the s2n_crypto_parameters structure instead of the connection. It doesn't change over the course of the handshake depending on the state of the connection. There's no initial vs secure version.

You might want to just make it a field on the s2n_connection object.